opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1644,6 +1698,13 @@ namespace Opm {
                     cellRes[ flowPhaseToEbosCompIdx( flowPhaseIdx ) ] /= refDens;
                     cellRes[ flowPhaseToEbosCompIdx( flowPhaseIdx ) ] *= cellVolume;
                 }
+                if (has_solvent_) {
+                    // no need to store refDens for all cells?
+                    const auto& intQuants = ebosSimulator_.model().cachedIntensiveQuantities(cellIdx, /*timeIdx=*/0);
```

Comment:
So, the usage here is *not a bug* even if it is something we want to move away from?